auth-rest-api/src/main/java/com/wipro/ats/bdre/md/auth/rest/AuthenticatorAPI.java

```diff
@@ -187,13 +191,12 @@ public String gotoPage(@PathVariable("page") String page) {
     }
 
     @RequestMapping(value = "/admin/sessions", method = RequestMethod.GET)
-    public
-    @ResponseBody
+
+    public @ResponseBody
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)